### DISCUSSION TITLE
Tutorial for devicescript

### DISCUSSION THREAD

**Discussion by AmmarAhmed1735** [microsoft/devicescript#418] [Q&A] [answered]
Looking for a devicescript implementation tutorial

**Comment by pelikhan** [comment, marked as answer]
You can start from the samples page (https://microsoft.github.io/devicescript/samples), each sample is written as a tutorial with a screen recording.

**Reply by georgik** [reply]
@pelikhan  URL does not work anymore: https://microsoft.github.io/devicescript/getting-started/samples - Page Not Found

**Reply by pelikhan** [reply]
Oh yes moved to /samples

**Reply by pelikhan** [reply]
https://microsoft.github.io/devicescript/samples
